### PR TITLE
chore: use dev-net chain config for local node

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Close: https://github.com/FuelLabs/fuels-ts/issues/2303

## Summary:
- Updated the 2 chains config to use the same values from the dev-net.
- Updated the base asset ID to be the same value as the dev-net.
- Updated funding amounts within our tests, as the calculated fee value increased.

### CODE REVIEW

File: .fuel-core/configs/chainConfig.json [hunk 1]
```diff
@@ -98,168 +98,168 @@
...
-              "base": 1645,
-              "units_per_gas": 14
+            "HeavyOperation": {
+              "base": 5254820,
+              "gas_per_unit": 0
             }
           }
         }
       },
       "base_asset_id": "0000000000000000000000000000000000000000000000000000000000000000",
```
Comment (Torres-ssf) [inline .fuel-core/configs/chainConfig.json:261]: @FuelLabs/sdk-ts Should we use the `base_asset_id` to the same ID used by the Testnet as well?

**Review by Dhaiwat10 — APPROVED** [review]
🚀

**Review by danielbate — APPROVED** [review]

**Comment by Torres-ssf** [comment]
@nedsalk I wonder if merging this PR now will result in breaking https://github.com/FuelLabs/fuels-ts/pull/1356.

**Comment by nedsalk** [comment]
@Torres-ssf I don't think it will break because only the values of the configs changed, not the interface.

**Review by arboleya — APPROVED** [review]

**Pull request merged**